contracts/mainnet/connectors/notional/main.sol

```diff
@@ -0,0 +1,671 @@
+pragma solidity ^0.7.6;
+pragma abicoder v2;
+
+/**
+ * @title Notional
+ * @dev Fixed Rate Lending and Borrowing
+ */
+
+import {Helpers} from "./helpers.sol";
+import {Events} from "./events.sol";
+import {DepositActionType, BalanceActionWithTrades, BalanceAction} from "./interface.sol";
+import {TokenInterface} from "../../common/interfaces.sol";
+
+abstract contract NotionalResolver is Events, Helpers {
+    /**
+     * @notice Deposit collateral into Notional, this should only be used for reducing risk of
+     * liquidation. 
+     *  @dev Deposits into Notional are not earning fixed rates, they are earning the cToken
+     * lending rate. In order to lend at fixed rates use `depositAndLend`
+     * @param currencyId notional defined currency id to deposit
+     * @param useUnderlying if true, will accept a deposit in the underlying currency (i.e DAI), if false
+     * will use the asset currency (i.e. cDAI)
+     * @param depositAmount amount of tokens to deposit
+     * @param getId id of depositAmount
+     * @param setId id to set the value of notional cash deposit increase (denominated in asset cash, i.e. cDAI)
+     */
+    function depositCollateral(
+        uint16 currencyId,
```

Comment:
Is there any API endpoint or docs which mentions all currencyID?

Reply:
Currency ID is autoincrementing from 1 to N and maps to a currency in the system. Currently: ETH is 1, DAI is 2, USDC is 3, WBTC is 4. You can get more details about each currency id from the subgraph:
https://thegraph.com/hosted-service/subgraph/notional-finance/mainnet-v2

Once set, the currency that is mapped to at a particular ID won't change.

Reply:
Alright, thanks